eth/core/processor.go

```diff
@@ -193,6 +194,8 @@ func (sp *StateProcessor) Finalize(
 		receipt.BlockHash = hash
 		for _, log := range receipt.Logs {
 			log.BlockHash = hash
+			log.Index = logIndex
```

Comment:
@calbera we good with this change?

Reply:
yea we need this one, weren't setting `Index` field. geth does it in `AddLog` bc they store logs per block instead of polaris per tx